pkg/pod/status_test.go

```diff
@@ -1276,6 +1519,11 @@ func TestMakeTaskRunStatus(t *testing.T) {
 			// Common traits, set for test case brevity.
 			c.want.PodName = "pod"
 			c.want.StartTime = &metav1.Time{Time: startTime}
+			for i := range c.want.TaskRunStatusFields.Steps {
+				if c.want.TaskRunStatusFields.Steps[i].Results == nil {
+					c.want.TaskRunStatusFields.Steps[i].Results = []v1.TaskRunResult{}
```

Comment:
Is the comment addressed? Do we still need this?

Reply:
Yes, I could not do it without this. The tests keep failing.

It's a nit so we can fix it later if we figure it out.